packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,520 @@
+
+Grid is built using the [Flex component](/utilities/flex) and can be made responsive.
+
+## Examples
+
+#### Equal column widths
+
+Use Column to lay out equal-sized content, such as a feed of cards.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph}} language="jsx">
+  {`<Grid gutter="space30">
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+#### Unequal column widths, using column span
+
+Use `span` to allow content to span multiple columns, such as a layout with a sidebar that takes up 1/4 of the screen and a content area that takes up 3/4 of the screen.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph}} language="jsx">
+  {`<Grid gutter="space30">
+  <Column span={8}>
+    <Heading as="h4" variant="heading40">Content Heading</Heading>
+    <Paragraph>
+      Kale chips distillery authentic, portland etsy cloud bread vinyl gentrify drinking vinegar viral meh hot
+      chicken bitters fashion axe palo santo. Chillwave fixie sustainable <i>helvetica</i> etsy.
+    </Paragraph>
+  </Column>
+  <Column span={4}>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+<Callout variant="warning">
+  <CalloutTitle as="h4" variant="heading40">An Important Note About Span and Offset</CalloutTitle>
+  <CalloutText>When using the <strong>span</strong> or <strong>offset</strong> props, base your layout off a 12-column grid. For example, if you want to have two columns with one spanning 8 columns, one column would span 8 and the other would need to span 4.</CalloutText>
+</Callout>
+
+#### Offset column
+
+Use `offset` to push a column to the right by a certain number of columns. For example, a layout with centered columns.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph}} language="jsx">
+  {`<Grid gutter="space30">
+  <Column span={8} offset={2}>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Card Heading</Heading>
+      <Paragraph>Chillwave fixie sustainable helvetica etsy.</Paragraph>
+    </Card>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+#### Stacked columns in a responsive view
+
+Use `vertical` to stack columns vertically instead of horizontally. Use this when your layout needs to respond to a smaller view.
```

Comment:
Could this example be slightly misleading? You call it a responsive view, but the layout doesn't actually respond to the browser. You're not using the responsive values in your props.

There's also no example showing me how to do responsive values

Reply:
hmmm I maybe miss understanding how responsive grid work, but I kind of expect

```
<Grid>
  <Column span={[12 , 6, 4]} />
  <Column span={[12 , 6, 4]} />
  <Column span={[12 , 12, 4]} />
</Grid>
```
to out put a 3 stack on mobile, 2 over 1 on the next breakpoint and 3 in a row after. But the 3 stacked doesn't work

Reply:
Because our site isn't responsive, its hard to show actual responsive examples. However I did rewrite the vertical section so it explains the vertical prop more. I also added a responsive column example that links to the storybook story: https://github.com/twilio-labs/paste/pull/307/files#diff-031c2d993f08f8fad55f849848f24b5cR231

As for your example, I didn't account for the 2 over 1 on a breakpoint use case. That is something we should add. I can add a ticket to our backlog.

In order to stack on mobile, you need the vertical prop set on `Grid`. Something like `vertical={[true, false, false]}` in your example. The true on vertical sets `flex-direction: column` so all columns will to go full with and stack.